packages/nodes-base/package.json

```diff
@@ -698,7 +698,7 @@
     "basic-auth": "^2.0.1",
     "change-case": "^4.1.1",
     "cheerio": "1.0.0-rc.6",
-    "chokidar": "^3.5.2",
+    "chokidar": "3.5.2",
```

Comment:
What is this change? Could you add a description why it's needed to the PR?

Reply:
To fix the build. Fixed it yesterday in `master`: https://github.com/n8n-io/n8n/pull/2696